### PR TITLE
feat(workspace): upgrade to Angular 12

### PR DESCRIPTION
**Pull request by jkubiszewski**
BREAKING CHANGE: Angular 12 support

### CODE REVIEW

File: projects/elements/karma.conf.js [hunk 1]
```diff
@@ -10,24 +10,38 @@ module.exports = function (config) {
...
+        { type: 'json' },
+      ],
     },
     reporters: ['spec'],
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['ChromeHeadless'],
+    browsers: ['Chrome'],
```
Comment (tomastrajan) [inline projects/elements/karma.conf.js:43]: We should stick to `'ChromeHeadless'`

File: projects/elements/ng-package.json [hunk 2]
```diff
@@ -2,6 +2,6 @@
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/elements",
   "lib": {
-    "entryFile": "src/public_api.ts"
+    "entryFile": "src/public-api.ts"
```
Comment (tomastrajan) [inline projects/elements/ng-package.json:5]: Was the file renamed too ?

File: projects/elements-demo/karma.conf.js [hunk 3]
```diff
@@ -10,24 +10,38 @@ module.exports = function (config) {
...
+        { type: 'json' },
+      ],
     },
     reporters: ['spec'],
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['ChromeHeadless'],
+    browsers: ['Chrome'],
```
Comment (tomastrajan) [inline projects/elements-demo/karma.conf.js:43]: We should stick to `'ChromeHeadless'`

**Comment by tomastrajan** [comment]
Hi @jkubiszewski ! Thank you for your PR! Looks great in general, just left couple of small comments ;) Looking forward to merging this soon!

File: projects/elements-demo/karma.conf.js (hunk 3 above)
Comment (jkubiszewski) [reply projects/elements-demo/karma.conf.js]: fixed

File: projects/elements/ng-package.json (hunk 2 above)
Comment (jkubiszewski) [reply projects/elements/ng-package.json:5]: yes :)

File: projects/elements/karma.conf.js (hunk 1 above)
Comment (jkubiszewski) [reply projects/elements/karma.conf.js]: fixed

**Pull request merged**